app/controllers/api/v1/targets_controller.rb

```diff
@@ -3,6 +3,10 @@
 module Api
   module V1
     class TargetsController < ApiController
+      before_action :targets, only: [:index]
```

Comment:
I'm not a huge fan of callbacks, I think that we could just move the logic from the `targets` method here, since this is the only place where it's needed.

Not a blocker, just want to know your thoughts about this 🙂

Reply:
We should avoid using callbacks as much as possible